src/snackbar/README.md

```diff
@@ -0,0 +1,64 @@
+# Snackbar
+
+## Introduction
+
+The Material Design Lite (MDL) **snackbar** component is a container used to notify a user of an operation's status. It displays at the bottom of the screen.
+
+Snackbars may also be used as a **toast** component. A toast may not be removed by the user and does not contain any actions.
+
+## Usage
+
+&nbsp;1. Add the `mdl-js-snackbar` class to the body.
+
+```html
+<body class="mdl-js-snackbar">
+</body>
+```
+
+&nbsp;2. Use JavaScript to trigger a snackbar.
+
+```JavaScript
+var data = {
+  message: 'Message Sent'
+};
+document.body.MaterialSnackbar.showSnackbar(data);
+```
+
+## Examples
+
+### Snackbar
+
+```JavaScript
+var data = {
+  message: 'Message Sent',
+  actionHandler: function(event) {},
+  actionText: 'Undo',
+  timeout: 10000
+};
+document.body.MaterialSnackbar.showSnackbar(data);
+```
+
+### Toast
+
+```JavaScript
+document.body.MaterialSnackbar.showSnackbar({message: 'Image Uploaded'});
+```
+
+## Data Object
+
+The Snackbar components `showSnackbar` method takes an object for snackbar data.
+The table below shows the properties and their usage.
+
+| Property | Effect | Remarks | Type |
```

Comment:
We don't do alignment with any other tables, so... /me pulls out lazy card and adds another hole-punch.

Unless we update the rest, I'm not worrying about it since if any content gets modified it can offset the alignment and then cause a mess keeping it in alignment. Probably wouldn't happen often, but still better to avoid the headace unless there is a good reason for it (since most people won't look at the raw markdown.)